src/components/Lottie/index.tsx

```diff
@@ -79,19 +79,20 @@ function Lottie({source, webStyle, shouldLoadAfterInteractions, ...props}: Props
         return unsubscribeNavigationBlur;
     }, [browser, navigationContainerRef, navigator]);
 
+    // If user is being navigated away, let pause the animation to prevent memory leak.
+    useEffect(() => {
+        if (!animationRef || !hasNavigatedAway) {
```

Comment:
`animationRef` always an object
```suggestion
        if (!animationRef.current || !hasNavigatedAway) {
```